### DISCUSSION TITLE
Add sources link of each GSI maintainer

### DISCUSSION THREAD

**Discussion by ghost** [phhusson/treble_experimentations#2404] [Ideas]
Some don't open their sources & just upload their GSI images & use telegram as forum chats. I don't like this behaviour.

[Under here](https://github.com/phhusson/treble_experimentations/wiki/Generic-System-Image-%28GSI%29-List), add a column named "Source" & link every gsi's build sources on it.

Closes #2403

**Comment by phhusson** [comment]
yeah sure feel free to do that change

Le jeu. 4 août 2022 à 06:55, Prince Cooper ***@***.***> a
écrit :

**Reply by ghost** [reply]
[Fixed now](https://github.com/phhusson/treble_experimentations/wiki/Generic-System-Image-(GSI)-List/_compare/4dd2937882eefc485a973d912c2b76a035125504...ee3f42fa3675d676b555486c10f4df6a0bf6f097)

Closed #2403
